src/Lachain.Core/Consensus/ConsensusManager.cs

```diff
@@ -326,5 +326,10 @@ public void Terminate()
         {
             _terminated = true;
         }
+
+        public EraBroadcaster GetEraBroadcaster()
+        {
+            return _eras[CurrentEra];
```

Comment:
It's possible that currentEra is null for a long period of time during the start of the machine. You can trigger it locally like this. 
(1) Start 4 nodes. Stop node-1 immediately.
(2) Wait till other blocks generate 100 blocks. 
(3) Start node-1. Now call "la_consensusState". The rpc call will throw exceptions. 

So, it's better to check if this _eras has "CurrentEra" or not and return null if it does not exist.